packages/zipkin-instrumentation-hapi/test/integrationTest.js

```diff
@@ -0,0 +1,152 @@
+const sinon = require('sinon');
+const {Tracer, ExplicitContext} = require('zipkin');
+const Hapi = require('hapi');
+const middleware = require('../src/hapiMiddleware');
+
+describe('hapi middleware - integration test', () => {
+  it.skip('should create traceId', () => {});
```

Comment:
@eirslett Let me know if you want it removed and I'll do it. Really looking forward to using your awesome library on my services!